charts/core/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: charts-core
 description: A Helm chart for Kubernetes
 type: application
-version: 2.1.0
```

Comment:
we never published 2.1.0 , but I need to change this file so the pipeline picks it up. I think it'd be better to release version 2.1.0 instead of 2.1.1 but I dont know how to make the pipeline use it

Reply:
2.1.0 is good